src/hooks/useEvent.ts

```
@@ -1,6 +1,8 @@
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable react-hooks/exhaustive-deps */
 import * as React from 'react';
 
-export default function useEvent<T extends Function>(callback: T): T {
+function useEvent<T extends Function>(callback: T): T {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)